gcn_email_service/helpers.py

```diff
@@ -0,0 +1,26 @@
+import logging
+import threading
+
+
```

Comment:
PEP8: there should be only one blank line between module-level statements that are not function or class declarations.

```suggestion
```